### ISSUE TITLE
DP.001.00 - Switch from all participants to only a subset of important ones

### ISSUE THREAD

**Issue by ryshoooo** [computerskole/cvr#2]
Only these should be included:
Direktion
Stiftere
Revision
Legale Ejere
Reelle Ejere

**Comment by Dennis87DK** [comment]
Can we make it into 2?
DP.001.001
1# Direktion, Revision, Reelle Ejere should be one, to catch the person in charge of an organisation. 
DP.001.002
2# Reelle Ejere, Legale Ejere, Revision

**Comment by ryshoooo** [comment]
Not a problem.

However remember that DP.001.001 and DP.001.002 are kind of only intermediary products, but of course can be used in some cases. The real product affected is DP.001.00.. but that's OK I can add the total numbers from each DP.001.001 and DP.001.002.. something like `totalEjereAktive` and `totalLegalEjereAktive` and similar for opløst and opløser.

Still though however there is an unresolved question #25 that kind of blocks this.

**Comment by Dennis87DK** [comment]
Yes #25 should also be effected. other wise we might have an issue. i want to ensure we show correct information, excluting happenings on the vat. like the sample i got 2000 bankruptcy, but only becourse he was in business creating them and selling "the startet company" so no real connection.
